docs/modules/pipelines/pages/job-placement-control.adoc

```diff
@@ -0,0 +1,25 @@
+= Jet Job Placement Control
+:description: Your Jet processing jobs can be distributed across a defined subset of the cluster. This approach provides finer control of your Jet processing, which means that you can distribute your workload to meet your requirements. 
+:page-enterprise: true
+
+{description}
+
+NOTE: Your license key must include `Advanced Compute` to activate this feature.
+
+For example, you can configure Jet processing jobs so that they run on lite members only, allowing you to split your computational and storage requirements without the need to configure each job separately. You control the members to use for your Jet job processing on a job-by-job basis.
+
+Distributing the processing job in this way allows you to find the best balance for your processing and storage requirements. Separating the processing from the data serving requirement means that less stress is put on your storage component's resources, as they only need to serve the data and not carry out any of the processing. This can help you to spread the load on your cluster across the members.
+
+Your storage components still use their resources to serve data. You must be sure that the processing element of the job uses considerably more resources than the data retrieval element before using job placement control in this manner. 
+
+To use job placement control, create and/or submit a job using the `JobBuilder` API, which you can use to configure the following:
+
+** The job configuration
+** The pipeline or DAG
+** The member selection criteria for the processing
+
+For further information on the `JobBuilder` API, refer to the link:https://docs.hazelcast.org/docs/latest/javadoc/com/hazelcast/jet/JetService.JobBuilder.html[API Reference, window=_blank].
+
+For further information on submitting an isolated job, see xref:pipelines:submitting-jobs.adoc#isolated-jobs[Submitting Jobs].
+
+For an example of an isolated job, see xref:architecture:distributed-computing.adoc[], or use the link:https://github.com/hazelcast/hazelcast-code-samples/tree/master/jet/wordcount-compute-isolation[provided code sample]. For further information on using our code samples, refer to the link:https://github.com/hazelcast/hazelcast-code-samples/blob/master/README.md[code samples ReadMe].
```

Comment:
This code sample is the only place where we still mention "compute isolation" - I think it's okay though because the example does use `LITE` members only, so it is an example of isolating your compute away form data members.